src/components/projects/projectsList.js

```diff
@@ -0,0 +1,64 @@
+// import { FaDiscord, FaEarthAmericas, FaGithub } from 'react-icons/fa6';
+
+const projectsList = [
+  {
+    title: 'RecordSponge',
+    description:
+      "RecordSponge is an app that helps community organizations quickly analyze an individual's criminal history to determine if they qualify to have their records expunged.",
+    status: 'Live',
+    logo: { image: '/assets/logoRecordSponge.svg', alt: 'RecordSponge logo' },
```

Comment:
logo can just be `logo: '/assets/logoRecordSponge.svg'` to simplify after removing alts